### PR TITLE
🔧 profile page buttons

### LINKED ISSUE #6167: Profile page - Icon alignment - Canary
### What happened?

Icons are not aligned on the profile page, also extends to mobile.

### Please reproduce in steps

Log in -> Profile

### Expected Behavior

Icons should be aligned

### What browsers are you seeing the problem on?

Desktop Firefox

### At which address did you encounter bug?

canary.kodadot.xyz

### Are you logged in?

Yes

### Which wallet you are using?

Talisman

### At which chain did you encounter bug?

_No response_

### Screenshots

Beta: 
image
Canary:
image
Mobile: 
image

### Relevant log output

_No response_

### Payment link for reward

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by roiLeo**
https://github.com/kodadot/nft-gallery/pull/6160#issuecomment-1580858233?
- closes #6167 
- replaced icon with sharp pack
- replaced invisible components
- removed dotscanner
- fixed `DonationButton`

Screenshot 2023-06-08 at 14-38-32 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer
Screenshot 2023-06-08 at 14-40-21 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer
Screenshot 2023-06-09 at 08-44-09 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer

### CODE REVIEW

**Comment by prury** [comment]
Clicking on them only opens the link if you click on the image, not in the whole button per se.

**Review by preschian — CHANGES_REQUESTED** [review]
doesnt work:
- gift button to 404 page
- qr code title

**Comment by roiLeo** [comment]
> doesnt work:
> 
>     * gift button to 404 page
> 
>     * qr code title

Thanks for the review, now fixed

Screenshot 2023-06-09 at 10-08-47 NFT Artist Profile on KodaDot KodaDot - NFT Market Explorer

**Review by preschian — APPROVED** [review]

**Comment by daiagi** [comment]
image

can we add some gap or remove the double borders?

**Comment by roiLeo** [comment]
> can we add some gap or remove the double borders?

Should be possible, but a bit long to debug. I think we might wait for the new design because they could disappear

**Comment by prury** [comment]
> > can we add some gap or remove the double borders?
> 
> Should be possible, but a bit long to debug. I think we might wait for the new design because they could disappear

**Comment by prury** [comment]
sorry, missclicked

**Comment by daiagi** [comment]
>Should be possible, but a bit long to debug. I think we might wait for the new design because they could disappear

fine by me, let's see what the new design brings

File: components/rmrk/Profile/ProfileDetail.vue [hunk 1]
```diff
@@ -559,6 +554,10 @@ export default class ProfileDetail extends mixins(
     }
   }
 
+  get realworldFullPath(): string {
+    return `${window.location.origin}${this.$route.fullPath}`
+  }
```
Comment (daiagi) [inline components/rmrk/Profile/ProfileDetail.vue:559]: smol

  ```suggestion
get realworldFullPath(): string {
    return window.location.href
  }
  ```

**Review by daiagi — APPROVED** [review]

**Pull request merged**